src/spoof.ts

```diff
@@ -87,15 +98,16 @@ const getElementBox = async (page: Page, element: ElementHandle, relativeToMainF
   }
 }
 
-export function path (point: Vector, target: Vector, spreadOverride?: number)
-export function path (point: Vector, target: BoundingBox, spreadOverride?: number)
-export function path (start: Vector, end: BoundingBox | Vector, spreadOverride?: number): Vector[] {
+export function path (point: Vector, target: Vector, spreadOverride?: number, moveSpeed?: number | null)
+export function path (point: Vector, target: BoundingBox, spreadOverride?: number, moveSpeed?: number | null)
+export function path (start: Vector, end: BoundingBox | Vector, spreadOverride?: number, moveSpeed?: number | null): Vector[] {
```

Comment:
I think at this point `path` should be turned into an `options` parameter. Too many positional arguments lead to unclear APIs and verbose function calls calls (you have one in this PR for example)
```ts
path(previous, rand, undefined, options?.moveSpeed)
```
instead it could be made turned into
```ts
path(previous, rand, {
  moveSpeed: options?.moveSpeed
})
```
It would have to be non-breaking ofc so the type signature would need to be something along the lines of
```ts
export function path (start: Vector, end: BoundingBox | Vector, moveSpeedOrOptions?: number | PathOptions) {
 // ..
}
```
Also I don't think `null` should be used here... or really anywhere in javascript :grimacing:

Reply:
I think you are right.